packages/backend/src/database/entities/dashboards.ts

```diff
@@ -24,6 +25,15 @@ type DbDashboardVersion = {
     created_at: Date;
 };
 
+type DbDashboardView = {
+    dashboard_view_id: number;
+    dashboard_version_id: number;
+    dashboard_view_uuid: string;
```

Comment:
what if we move to `uuid` as primary keys? we've paid the price for too long from that old design choice haha